modules/rosa-sts-overview-of-the-default-cluster-specifications.adoc

```diff
@@ -70,6 +76,12 @@ ifndef::rosa-hcp[]
 endif::rosa-hcp[]
 ifdef::rosa-hcp[]
 * Default AWS region for installations using the ROSA CLI (`rosa`): Defined by your `aws` CLI configuration
+* Default EC2 IMDS endpoint v2 is enabled
```

Comment:
@bchandra-ocp I guess it would be nice to write "Default EC2 IMDS endpoints (both v1 and v2) are enabled" cc: @bmcelvee, Here too

Reply:
It's super important that we are specific that optional is what we're turning on. I think we could say something like "EC2 Instance Metadata Service (IMDS) is enabled and allows the use of IMDSv1 or IMDSv2 (token optional)".

Reply:
Done